swiper.el

```diff
@@ -816,7 +816,7 @@ When non-nil, INITIAL-INPUT is the initial search pattern."
                    (if initial-input
                        (cl-position-if
                         (lambda (x)
-                          (= (1+ preselect) (swiper--line-number x)))
+                          (< (1+ preselect) (swiper--line-number x)))
```

Comment:
> As can be in the change, if we modify this from equal to less than, it takes the first candidate with a line number greater than the current point.

Shouldn't we check `<=` instead of `<`, to handle a match at point?  Otherwise the regression test `swiper-thing-at-point` will fail.

Reply:
Yes, you are correct. I had debated between the two, but had gotten used to the behavior of "<" which will easily cycle through matches in a file without having to go to the next item in the list. However, "<=" would be more consistent with previous behavior.